src/services/reviews.js

```diff
@@ -713,6 +715,72 @@ module.exports = class reviewsHelper {
 			if (process.env.CONSUMPTION_SERVICE != common.SELF) {
 				//resource creation through kafka
 				if (process.env.RESOURCE_KAFKA_PUSH_ON_OFF == common.KAFKA_ON) {
+					if (resourceData?.type == common.ROLLOUT_TYPE_PROGRAM) {
+						let rolloutId = null
+						if (resourceData?.published_id) {
+							// check for rollout to fetch id and org id
+							const fetchRollout = await rolloutQueries.findOne(
```

Comment:
@adithyadinesh0412 where we are solution rollout for example when adding a new resource we need to create solution rollout and map to program rollout